integrationtests/selector/integrationtest_selector.py

```diff
@@ -297,9 +333,9 @@ def test_newdata() -> None:
         "config": {"limit": -1, "reset_after_trigger": True},
     }
 
-    pipeline_id = selector.register_pipeline(
-        RegisterPipelineRequest(num_workers=2, selection_strategy=JsonString(value=json.dumps(strategy_config)))
-    ).pipeline_id
+    grpc_handler = get_grpc_handler()
```

Comment:
done. register_pipeline is moved to Supervisor. Since a Supervisor is created per pipeline, it is not straightforward to reuse the same Supervisor for all pipeline configs in the tests. I copied the register_pipeline logic to the integration test utils.